tests/gis_tests/geos_tests/test_io.py

```diff
@@ -35,7 +35,7 @@ def test01_wktreader(self):
 
     def test02_wktwriter(self):
         # Creating a WKTWriter instance, testing its ptr property.
-        wkt_w = WKTWriter()
+        wkt_w = WKTWriter(trim=False)
```

Comment:
This seems unnecessary as `trim=False` is the default value.

Reply:
fixed :-)